cookiecutter.json

```diff
@@ -40,7 +42,8 @@
         "license": "Select a license",
         "backend": {
             "__prompt__": "Choose a build backend",
-            "hatch": "Hatchling (recommended for pure Python projects)"
+            "hatch": "Hatchling (recommended for pure Python projects)",
+              "hatch_vcs": "Enable hatch version control (vcs)?"
```

Comment:
```suggestion
            "hatch": "Hatchling (recommended for pure Python projects)",
            "hatch_vcs": "Enable hatch version control (vcs)?"
```

Reply:
Is this prompt offered to the users once they select `hatch`? I think you are offering the `hatch-vcs` option under `backend`? Users should select the hatch backend first and then a prompt saying "Enable VCS versioning? (Yes/No)" should enable/disable `hatch-vcs`.

Reply:
Instead of `hatch_vcs`, just call it `vcs`. This will make it easy to scale with potential new build backends in the future.

Reply:
> Is this prompt offered to the users once they select `hatch`? I think you are offering the `hatch-vcs` option under `backend`? Users should select the hatch backend first and then a prompt saying "Enable VCS versioning? (Yes/No)" should enable/disable `hatch-vcs`.

Yes I was trying to do that and I couldn't find any documentation of nested conditional choices for cookiecutter, though I have gotten it figured out for copier.

Reply:
Ah no, not necessarily nested, just an independent "vcs" prompt after "hatch" is selected. Treat the build backend and vcs as two independent things, if that makes sense. They should just be prompted to the user in the order 1. backend 2. vcs.

This way, when we add new build backends, say setuptools, the vcs option will update the generated code with setuptools_scm instead of hatch-vcs.